collector.go

```diff
@@ -59,6 +59,24 @@ var (
 		"Atmospheric pressure measurement in mb",
 		varLabels,
 		nil)
+
+  windDesc = prometheus.NewDesc(
+    prefix+"wind_kmh",
```

Comment:
I wonder if we should have this in "meters/second" instead so that it is an SI unit?

`wind_speed_<unit>` would be a better fit, I think.